src/context/auto-executor.ts

```diff
@@ -363,15 +364,17 @@ export class AutoExecutor {
     }
 
     const result = await this.context.qstashClient.batchJSON(
-      steps.map((singleStep) => {
+      steps.map((singleStep, index) => {
+        const lazyStep = lazySteps[index];
         const { headers } = getHeaders(
           "false",
           this.context.workflowRunId,
           this.context.url,
           this.context.headers,
           singleStep,
           this.context.failureUrl,
-          this.context.retries
+          this.context.retries,
```

Comment:
What is the difference between `context.retries` and `lazyStep.retries`?

Reply:
context.retries is the retry configuration of the whole workflow

lazyStep.retries only exists in LazyCallStep and designates the retry to use in the context.call

The parameters of the getHeaders method is slightly out of control, but we are thinking of updating it.